json_rpc/http.md

```diff
@@ -0,0 +1,243 @@
+# JSON RPC HTTP Methods
+
+The block engine accepts HTTP requests using the [JSON-RPC 2.0](https://www.jsonrpc.org/specification) specification.
+
+## RPC HTTP Endpoint
+
+**Default port:** 443 e.g. [https://mainnet.block-engine.jito.wtf:443/api/v1/bundles](https://mainnet.block-engine.jito.wtf:443/api/v1/bundles), [https://{REGION}.mainnet.block-engine.jito.wtf:443/api/v1/bundles](https://${REGION}.mainnet.block-engine.jito.wtf:443/api/v1/bundles)
+
+## Request Formatting
+
+To make a JSON-RPC request, send an HTTP POST request with a `Content-Type: application/json` header. The JSON request data should contain 4 fields:
+
+- `jsonrpc`: `string` - set to "2.0"
+- `id`: `number` - a unique client-generated identifying integer
+- `method`: `string` - a string containing the method to be invoked
+- `params`: `array` - a JSON array of ordered parameter values
+
+We follow the same request formatting as Solana json rpc requests. You can find the documentation [here](https://docs.solana.com/api/http#request-formatting)
+
+## Authorization
+
+In the short-term we don't require authentication to send the requests.
+
+If there are any changes to the authentication mechanism it would be updated in the document and communicated with all the stakeholders.
+
+## Definitions
```

Comment:
can you define bundle here?

Reply:
bundle is defined as 
`
- Bundle: Bundles are a list of transactions that execute sequentially and atomically. “All or nothing” so to speak. This means that a user can send a bundle that contains multiple transactions and guarantee that they are all executed one after the other and the bundle succeeds only if all individual transactions succeed.
`